src/Microsoft.AspNetCore.Server.HttpSys/AuthenticationHandler.cs

```diff
@@ -91,13 +81,23 @@ public Task ChallengeAsync(ChallengeContext context)
             return TaskCache.CompletedTask;
         }
 
-        public void GetDescriptions(DescribeSchemesContext context)
+        public Task<bool> HandleRequestAsync()
```

Comment:
Yep this PR was created against Iteration 2 (prior to named changes and the split of HandleRequest)

I'll update this PR now against latest